client/src/Components/CommentContainer/index.js

```diff
@@ -1,40 +1,47 @@
-import React, { useState } from 'react';
-// import { List } from 'antd';
+import React, { useState, useEffect } from 'react';
+import { List } from 'antd';
 import PropTypes from 'prop-types';
 import Comment from '../Common/Comment';
 import MainButton from '../Common/MainButton';
 
-import './style.css';
-
 const CommentContainer = ({ data, isAdmin }) => {
+  const [commentsToShow, setCommentsToShow] = useState([]);
+  const [next, setNext] = useState(3);
   const commentsPerPage = 3;
-  const [limit, setLimit] = useState(commentsPerPage);
+
+  const loopWithSlice = (start, end) => {
+    const slicedComments = data.slice(start, end);
+    setCommentsToShow([...commentsToShow, ...slicedComments]);
+  };
 
   const handleShowMoreComments = () => {
-    setLimit(commentsPerPage + limit);
+    loopWithSlice(next, next + commentsPerPage);
+    setNext(next + commentsPerPage);
   };
 
+  useEffect(() => {
+    loopWithSlice(0, commentsPerPage);
+  }, []);
+
   return (
     <div>
-      <ul className="list">
-        {data.length !== 0 ? (
-          data.slice(0, limit).map((item) => (
-            <li key={item.id}>
-              <Comment
-                isAdmin={isAdmin}
-                userName={item.user_name}
-                commentText={item.comment}
-                date={item.create_at}
-                rateValue={item.rating}
-              />
-            </li>
-          ))
-        ) : (
-          <Comment description="لا يوجد رياض أطفال" />
+      <List
+        split={false}
+        itemLayout="horizontal"
+        dataSource={commentsToShow}
+        renderItem={(item) => (
+          <List.Item>
```

Comment:
```suggestion
          <List.Item key="should be unique string">
```
source: https://ant.design/components/list/
to get more understanding about the key in React have a look here: https://reactjs.org/docs/lists-and-keys.html